src/components/Restoration/BusinessName.vue

```diff
@@ -160,11 +160,10 @@ export default class BusinessName extends Mixins(CommonMixin, DateMixin, NameReq
    * @returns a promise to return the NR, or throws a printable error
    */
   async fetchAndValidateNr (nrNum: string, phone: string, email: string): Promise<NameRequestIF> {
-    const nameRequest = await LegalServices.fetchNameRequest(nrNum)
+    const nameRequest = await LegalServices.fetchValidContactNr(nrNum, phone, email)
     if (!nameRequest) throw new Error('Error fetching Name Request')
```

Comment:
Please display specific errors if the fetch failed due to a missing or incorrect email/phone/whatever.

As line 163 is an await statement, you can add `.catch(error => { ...` to handle the response codes.

Reply:
I see you're handling the errors in fetchValidContactNr() -- perfect, thanks!